server/config/runtime.exs

```diff
@@ -225,10 +225,10 @@ if Tuist.Environment.env() not in [:test] do
       [
         scheme: "#{s3_scheme}://",
         host: s3_endpoint_host,
-        bucket_as_host: Tuist.Environment.s3_virtual_host(),
+        bucket_as_host: Tuist.Environment.s3_bucket_as_host(),
```

Comment:
These are unrelated to Minio, and they didn't cause any issues in production because they both have `true` as value.